### PR TITLE
Inference

### PR DESCRIPTION
**Pull request by lmaxwell**
Implement inference code for downstream models like audioset and audioset_strong.

### CODE REVIEW

**Comment by lifeiteng** [comment]
Labels 标签对 display name 是啥？ 我用 https://github.com/Audio-WestlakeU/audiossl/issues/12#issuecomment-2217406872 `mid_to_display_name.tsv` 出来的结果不太对

161722056537_ pic

File: audiossl/methods/atstframe/downstream/Inference_audioset_strong.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+                output.append(output_chunk)
+        output=torch.cat(output,dim=1)
+        output = self.head(output)
+        return output
+
+
+if __name__ == "__main__":
+    import sys
+    ckpt_path  = sys.argv[1]
+    m = InferenceAudioSetStrong(ckpt_path)
```
Comment (lifeiteng) [inline audiossl/methods/atstframe/downstream/Inference_audioset_strong.py:90]: `m.eval()` to disable dropout

**Comment by SaoYear** [comment]
Hi Feiteng, thank you for the interest and sorry for the mess on this part. I forgot to use the sorted label in the label generation codes. You could try this file for the provided checkpoint, we will fix the commited codes later.
[common_labels.txt](https://github.com/user-attachments/files/16423596/common_labels.txt)

**Comment by lmaxwell** [comment]
@Angelalilyer

File: audiossl/methods/atstframe/downstream/Inference_audioset_strong.py
Comment (Square-of-W) [inline audiossl/methods/atstframe/downstream/Inference_audioset_strong.py:1]: Would you mind explaining the reason why chunk_len in line 63 of this code was chosen to be 1001 instead of 1000?
